docs/releases/1.22-NOTES.md

```diff
@@ -0,0 +1,21 @@
+## Release notes for kOps 1.22 series
+
+**&#9888; kOps 1.22 has not been released yet! &#9888;**
+
+This is a document to gather the release notes prior to the release.
+
+# Significant changes
+
+# Breaking changes
+
+* Support for Kubernetes versions 1.15 and 1.16 has been removed.
+
+# Required Actions
+
+* To support [Node Termination Handler's Queue Process mode](/addons/#node-termination-handler), AWS cluster deletion now requires the kops CLI have `sqs:ListQueues` and `events:ListRules` permissions regardless of whether or not the addon is used.
+
+# Deprecations
+
+* Support for Kubernetes versions 1.17 is deprecated and will be removed in kOps 1.23.
```

Comment:
Should we also remove 1.18? It will be EOL in a few days.

Reply:
We can decide to do that anytime before we ship 1.22. Right now the code deprecates only 1.17. The last time we discussed this in Office Hours it was suggested we only deprecate one version this round.

Reply:
OK